lib/easypost/services/address.rb

```diff
@@ -2,6 +2,8 @@
 
 # Address objects are used to represent people, places, and organizations in a number of contexts.
 class EasyPost::Services::Address < EasyPost::Services::Service
+  CLASS = EasyPost::Models::Address
```

Comment:
Let's call it MODEL_CLASS so as to not confuse it with the class `self`

Reply:
resolved